### PR TITLE
feat: add amIsBefore and amIsAfter pipes for #118

### PR DESCRIPTION
**Pull request by StickNitro**
specs included for both pipes and added example to README

### CODE REVIEW

File: src/moment.module.ts [hunk 1]
```diff
@@ -29,11 +30,17 @@ const ANGULAR_MOMENT_PIPES = [
...
-  ParseZonePipe
+  ParseZonePipe,
+  IsBeforePipe,
+  IsAfterPipe
 ];
 
 @NgModule({
-  declarations: ANGULAR_MOMENT_PIPES,
-  exports: ANGULAR_MOMENT_PIPES
+  declarations: [
```
Comment (urish) [inline src/moment.module.ts]: What's the reason for this change?

Comment (StickNitro) [reply src/moment.module.ts]: No reason other than my IDE screwed up the module file when adding the pipes as it injects the declarations into the `declarations: []` section of the module

Comment (StickNitro) [reply src/moment.module.ts]: I can revert this as part of this PR if you prefer

Comment (urish) [reply src/moment.module.ts]: Please do

Comment (StickNitro) [reply src/moment.module.ts]: Done

Comment (urish) [reply src/moment.module.ts]: Thank you!

**Comment by urish** [comment]
Released as 3.4.0
